sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientTestsLive.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.Testing;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class TextAnalyticsClientTestsLive : RecordedTestBase
+    {
+        public const string EndpointEnvironmentVariable = "TEXT_ANALYTICS_ENDPOINT";
+        public const string SubscriptionKeyEnvironmentVariable = "TEXT_ANALYTICS_SUBSCRIPTION_KEY";
+
+        public TextAnalyticsClientTestsLive(bool isAsync) : base(isAsync, RecordedTestMode.Live) { }
+
+        public TextAnalyticsClient GetClient()
+        {
+            return InstrumentClient
+                (new TextAnalyticsClient(
+                    new Uri(Recording.GetVariableFromEnvironment(EndpointEnvironmentVariable)),
+                    Recording.GetVariableFromEnvironment(SubscriptionKeyEnvironmentVariable),
+                    Recording.InstrumentClientOptions(new TextAnalyticsClientOptions())));
+        }
+
+        [Test]
+        public void ConvertToDocumentInputTest()
```

Comment:
I think this can be a mock test, as it doesn't call the service.

Reply:
Yeah, you are right. I'll move it